readthedocs/templates/base.html

```diff
@@ -192,8 +195,7 @@ <h4>{% trans 'Business Info' %}</h4>
               <a href="https://readthedocs.com">{% trans 'Read the Docs for Business' %}</a>
             </li>
             <li>
-              {% url "donate" as donate_url %}
-              <a href='{{ donate_url }}'>{% trans 'Supporters' %}</a>
+              <a href='http://status.readthedocs.com/'>{% trans 'Site Status' %}</a>
```

Comment:
Hrm, the HTTPS version is missing a certificate. We should get one, probably.

Reply:
We can't get one unless we proxy it via something we control, unfortunately. I had it on a domain proxied by Cloudflare which worked, but doesn't for .com